types/sinon/ts3.1/index.d.ts

```diff
@@ -133,44 +135,24 @@ declare namespace Sinon {
          */
         lastArg: any;
 
+        /* tslint:disable:no-unnecessary-generics */
+
         /**
          * Returns true if the spy call occurred before another spy call.
          * @param call
+         *
          */
-        calledBefore(call: SinonSpyCall): boolean;
+        calledBefore<TSpyArgs extends any[]>(call: SinonSpyCall<TSpyArgs>): boolean;
```

Comment:
```suggestion
        calledBefore(call: SinonSpyCall<any>): boolean;
```

You can just pass `any` instead of `any[]` (or the default, which is also `any[]`) and then you wouldn't need to disable the tslint rule.